### DISCUSSION TITLE
Courseplay als "GPS" nutzen ohne Helfer

### DISCUSSION THREAD

**Discussion by LanDei83** [Courseplay/Courseplay_FS22#1984] [Q&A] [answered]
Hallo,
wäre es eigentlich möglich Courseplay als "GPS" zu nutzen wie es heute auch in der Landwirtschaft möglich ist?

Die Idee dahinter ist das wir auf einen Multiplayer Server eigentlich keine Helfer möchten und unsere krummen Felder nicht für normales GPS-Mod geeignet sind.

Daher meine Frage gibt es die Möglichkeit CP so zu ändern daß es beim Verlassen des Traktor dieser nicht weiter fährt?

Zusätzlich wäre es auch noch schön wenn es einmalig pro Tracktor gekauft werden muss.

Vielen Dank für eure Rückmeldung
Mit freundlichen Grüßen
Daniel

**Comment by Tensuko** [comment, marked as answer]
Das ist nicht mal eben so möglich und auch nicht in unserem Interesse. Sry.
